components/sections/ProvingTeamsSection.tsx

```diff
@@ -49,7 +49,7 @@ const ProverTeamsSection = async () => {
   )
 
   return (
-    <Card className="!p-0 !pb-6 md:!pb-8">
+    <Card variant="borderGradient" className="!p-0 !pb-6 md:!pb-8">
```

Comment:
I'd add this style as the default in the Card component. I think all of our cards are using this style.